src/easydmp/plan/management/commands/move_answer_data_to_answerset.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+from django.core.management import BaseCommand
+
+from easydmp.dmpt.models import Question
+from easydmp.plan.models import Plan, AnswerSet
+
+#
+# This was part of a refactor to change ownership of answer data, and may be deleted later.
+#
+
+def handle_plan(plan: Plan, delete: bool):
+    for k, d in dict(plan.data).items():
+        question = Question.objects.get(pk=int(k))
+        answerset = AnswerSet.objects.get(section=question.section)
+        answerset.data[k] = d
+        answerset.save()
+        if delete:
+            del plan.data[k]
+            plan.save()
+
+class Command(BaseCommand):
+    help = "Distribute answer data from Plan to appropriate AnswerSet"
+
+    def add_arguments(self, parser):
+        parser.add_argument('-p', '--plan', nargs='*', type=int, default=[],
+                            help='Plans to process (id). Default is all plans.')
+        parser.add_argument('-d', '--delete', type=bool, default=False,
+                            help='Whether to delete the answer data from Plan after it has been written to AnswerSet. Default is false.')
+
+    def handle(self, *args, **options):
+        plan_ids = options['plan']
+        if plan_ids:
+            plans = Plan.objects.filter(pk__in=plan_ids)
+        else:
```

Comment:
This being the default is a bit scary. In other upgrading management commands I usually solve this by an "all"-argument, no flags being a noop.

In this case you could either make mutually exclusive groups or change plan into a positional argument.